docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -193,6 +193,11 @@ If you do not want to use the HTTP feature then you can simply run the `remote-d
 In this mode the command will continuously rebuild the local application, so you can use an external tool such as odo or
 rsync to sync to the remote application.
 
+NOTE: It is recommended you use SSL when using remote dev mode, however even if you are using an unencrypted connection
+your password is never sent directly over the wire. For the initial connection request the password is hashed with the
+initial state data, and subsequent requests hash it with a random session id generated by the server and any body contents
```

Comment:
does that not mean if I stop the remote dev client side, do some edits and start again that I can't authenticate or do I misunderstand ?

Reply:
It creates a new session in that case.